src/Renderer/RendererOpenGL.cpp

```diff
@@ -829,18 +830,6 @@ void fillTextureArray(GLfloat x, GLfloat y, GLfloat u, GLfloat v)
 	TEXTURE_COORD_ARRAY[10] = static_cast<GLfloat>(x), TEXTURE_COORD_ARRAY[11] = static_cast<GLfloat>(y);
 }
 
-
-/**
- * The following code was developed by Chris Tsang and lifted from:
- *
- * http://www.codeproject.com/KB/openGL/gllinedraw.aspx
- *
- * Modified: Removed option for non-alpha blending and general code cleanup.
- *
- * This is drop-in code that may be replaced in the future.
- */
-static inline float _ABS(float x) { return x > 0 ? x : -x; }
-
```

Comment:
I don't think this comment pertains to the `_ABS` method alone, but also to the code that follows it.